docs/src/pass_flow.txt

```diff
@@ -0,0 +1,22 @@
+title Kong OIDC Plugin - "pass" functionality
```

Comment:
this file has a .txt extension - is it plantuml?  mermaid?  shouldn't it have a corresponding file extension?  or does github expect .txt?

Reply:
addressed, changed extension